### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by Prateekjain09**
Model containing 9 joints

### CODE REVIEW

**Comment by Prateekjain09** [comment]
Resolve conflicts button is not activated

File: billi_sim/launch/default.launch [hunk 1]
```diff
@@ -21,8 +21,20 @@
...
+respawn="false" output="screen">
+    <remap from="/joint_states" to="/billi/joint_states" />
+</node>
+
+<node pkg="rosbag" type="play" name="rosbag_play" output="screen" required="true"
+         args="--clock /home/prateek/practice_ws/src/billi_sim/bags/hyq.bag"/>
+        
+	
     </group>
-</launch>
```
Comment (pushpanshu0501) [inline billi_sim/launch/default.launch:28]: *

File: billi_sim/launch/spawn_billi.launch [hunk 2]
```diff
@@ -28,4 +28,4 @@
     respawn="false" output="screen">
    </node>
 
-</launch>
```
Comment (pushpanshu0501) [inline billi_sim/launch/spawn_billi.launch:31]: *

File: billi_sim/launch/default.launch [hunk 3]
```diff
@@ -21,8 +21,20 @@
         <include file="$(find billi_sim)/launch/spawn_billi.launch">
             <arg name="ugv_name" default="$(arg ugv_name)"/>
         </include>
-        <!-- <rosparam command="load" file="$(find billi_sim)/cfg/control_params.yaml"/> -->
```
Comment (pushpanshu0501) [inline billi_sim/launch/default.launch:24]: **

File: billi_sim/launch/default.launch [hunk 4]
```diff
@@ -21,8 +21,20 @@
...
-        <!-- <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="default_controller"/> -->
```
Comment (pushpanshu0501) [inline billi_sim/launch/default.launch:25]: same

File: billi_sim/launch/default.launch [hunk 5]
```diff
@@ -21,8 +21,20 @@
...
-        <!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/> -->
```
Comment (pushpanshu0501) [inline billi_sim/launch/default.launch:26]: same

File: billi_sim/CMakeLists.txt [hunk 6]
```diff
@@ -5,4 +5,4 @@ find_package(catkin_simple REQUIRED)
 catkin_simple()
 
 cs_install()
-cs_export()
```
Comment (pushpanshu0501) [inline billi_sim/CMakeLists.txt:8]: *

**Pull request closed without merging**